dotcom-rendering/src/components/AdSlot.tsx

```diff
@@ -275,18 +275,35 @@ const mostPopAdStyles = css`
 	position: relative;
 	min-height: ${adSizes.mpu.height + labelHeight}px;
 	min-width: 300px;
-	width: 300px;
+	max-width: 300px;
```

Comment:
v.minor but is 300px referring to `adSizes.mpu.width` ?

Reply:
Yes it is, good spot, I'll update to use `adSizes.mpu.width`